internal/service/common/certificatesigningrequest.go

```diff
@@ -0,0 +1,63 @@
+package common
```

Comment:
I don't foresee reuse here.  Why not move the code to the relevant functions in internal/service/certificatesigningrequest.go?